src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -340,20 +340,24 @@ public void clearEntries() {
     }
 
     public void appendNodes(List<FlutterLogEntry> entries) {
-      ApplicationManager.getApplication().invokeLater(() -> {
+      if (treeTable == null || uiThreadAlarm.isDisposed()) {
+        return;
+      }
+
+      uiThreadAlarm.cancelAllRequests();
+      uiThreadAlarm.addRequest(() -> {
```

Comment:
I don't know enough about how this work, but if we cancel older requests, will the work they do to append `entries` not get scheduled?

Reply:
Right.  Luckily it won't matter.  The simple implementation is non-incremental; each reload fetches the entries fresh.